### PR TITLE
Implement health check get and save apis

### PR DESCRIPTION
**Pull request by tapishr**
- [ ] Test ready
- [ ] Prod ready

### CODE REVIEW

File: pages/api/docs/getDpuHealth.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { getServerSession } from "next-auth/next";
+import { authOptions } from "../auth/[...nextauth]";
+import rudderStackEvents from "../events";
+import { getHealthStatusFromDB } from "../../../utils/db/healthStatus";
+
+const getDpuHealth = async (req: NextApiRequest, res: NextApiResponse) => {
+	const session = await getServerSession(req, res, authOptions);
+	if (!session || !session.user.id) {
```
Comment (MuskanPaliwal) [inline pages/api/docs/getDpuHealth.ts:9]: there might be a possibility that session does have use object. so we need to check for user object too.

Comment (MuskanPaliwal) [reply pages/api/docs/getDpuHealth.ts:9]:
```suggestion
	if (!session || !session.user?.id) {
```

File: pages/api/dpu/health.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+const healthHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+    console.info("[healthHandler] Saving dpu health info...");
+	const jsonBody = req.body;
+
```
Comment (MuskanPaliwal) [inline pages/api/dpu/health.ts:6]: This is not implemented yet right?

Comment (tapishr) [reply pages/api/dpu/health.ts:6]: Did it in `mkp/docs/test`

**Pull request merged**